accounts/serializers.py

```diff
@@ -10,11 +10,6 @@ class Meta:
             "id",
             "first_name",
             "last_name",
-            "is_staff",
-            "is_active",
-            "email",
-            "last_login",
-            "date_joined",
```

Comment:
this serializer didn't have a test, added [this ticket](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=12335f7f7fde4940975eab1ca1248d8a) to address later